File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseTimeFilterPostcodeDistrict.scala

package org.openapitools.client.model


case class ResponseTimeFilterPostcodeDistrict (
    _code: String,
    _properties: ResponseTimeFilterPostcodeDistrictProperties
)
object ResponseTimeFilterPostcodeDistrict {
    def toStringBody(var_code: Object, var_properties: Object) =
        s"""
        | {
        | "code":$var_code,"properties":$var_properties
        | }
        """.stripMargin
}

File: kotlin-spring/src/main/kotlin/org/openapitools/model/ResponseTimeFilterPostcodeDistrict.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.ResponseTimeFilterPostcodeDistrictProperties
import org.openapitools.model.String
import javax.validation.constraints.DecimalMax
import javax.validation.constraints.DecimalMin
import javax.validation.constraints.Max
import javax.validation.constraints.Min
import javax.validation.constraints.NotNull
import javax.validation.constraints.Pattern
import javax.validation.constraints.Size

/**
 * 
 * @param code 
 * @param properties 
 */
data class ResponseTimeFilterPostcodeDistrict (

        @get:NotNull 
        @JsonProperty("code") val code: String,

        @get:NotNull 
        @JsonProperty("properties") val properties: ResponseTimeFilterPostcodeDistrictProperties
) {

}

